tuna/miopen/driver/base.py

```diff
@@ -167,10 +168,10 @@ def insert_tensor(self, tensor_dict: dict) -> int:
         tid.valid = 1
         key = build_dict_val_key(tid)
         #cache the tensor table to avoid queries
-        if not DriverBase.tensor_id_map:
```

Comment:
once you move tensor_id_map inside __init__ they will become object attributes not class attributes. @cderb  is there a reason to keep them as class attr?

Reply:
It would be optimal to keep tensor_id_map as a class static or a global. This reduces the number of queries to produce the map. As it is, it is only re-queried when there is an integrity error (a miss) when accessing the map. Making it a object attribute would mean that this query would happen for each object individually (which would be slower).

Reply:
fine with me